readme.md

```diff
@@ -1,16 +1,13 @@
-<p class="github-only">
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-16-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```

Comment:
I have removed this. Did we need it?

Reply:
Nope, this is ok. Not sure how this ended up here.